Add recipe for git-bug

WillForan opened the pull request:
### Brief summary of what the package does
A minimal interface to [git-bug](https://github.com/git-bug/).

This package provides a completing-read menu to match existing bugs and another menu to act on a bug.

### Direct link to the package repository

https://github.com/WillForan/emacs-git-bug

### Your association with the package
Submitted by maintainer.

### Checklist

<!-- Please confirm by replacing `[]` with `[x]`: -->

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] I've used `M-x checkdoc` to check the package's documentation strings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

<!-- After submitting, please fix any problems the CI reports. -->

riscy commented:
Thanks for this.  Quick first pass -

⸺ `git-bug.el` with [melpazoid](https://github.com/riscy/melpazoid):
```
- `;;; Commentary` is much wider than 80 characters
- git-bug.el#L167: Lighter should start with a space
```

⸺ Package and license:
- git-bug.el needs *formal* license boilerplate and/or an [SPDX-License-Identifier](https://spdx.dev/ids/)

WillForan commented:
eek! Sorry I thought those were already resolved (promise I didn't just click the checkboxs). Hopefully fixed now.

Added melpazoid to `emacs-git-bug` and [see no more issues](https://github.com/WillForan/emacs-git-bug/actions/runs/17534080720/job/49794833395#step:5:28)

WillForan commented:
Thanks for spending so much time with this code! Very much appreciate the tips and typo catches.

Fixed the typo and removed markup from. FWIW, I've tried to setup org-babel+elisp to copy the commentary into the readme -- hoping that'd make it easier to keep them in sync. But I hadn't thought about how markup would be annoying in the source. My would-be-elegant doc-syncing is now regex soup :)
